client/src/components/board/CreateList.js

```diff
@@ -1,60 +1,60 @@
-import React, { useRef, useState, useEffect } from 'react';
-import { useDispatch } from 'react-redux';
-import { addList } from '../../actions/board';
-import { TextField, Button } from '@material-ui/core';
+import React, {useRef, useState, useEffect, useContext} from 'react';
+
+import {TextField, Button} from '@material-ui/core';
 import CloseIcon from '@material-ui/icons/Close';
+import {BoardContext} from "../../contexts/boardStore";
 
 const CreateList = () => {
-  const [adding, setAdding] = useState(false);
-  const [title, setTitle] = useState('');
-  const dispatch = useDispatch();
+    const [adding, setAdding] = useState(false);
+    const [title, setTitle] = useState('');
+    const {addList} = useContext(BoardContext);
```

Comment:
another thing i have noticed is that you haven't used an HOC to pass context. why is that?

Reply:
I read online that it is not preferred to use HOC now due to the simplicity introduced by UseContext Hook.